src/standardMethods.ts

```diff
@@ -306,19 +324,22 @@ export function generateRetrieveLatestEndpoint<RequestParams, ResponseType>(
 export function generateInsertEndpoint<RequestParams>(
   axiosInstance: AxiosInstance,
   resourcePath: string,
-  metadataMap: MetadataMap
+  metadataMap: MetadataMap,
+  chunkFunction?: (items: RequestParams[]) => RequestParams[][]
 ) {
   return async function insert(items: RequestParams[]): Promise<{}> {
-    const response = await rawRequest<ItemsResponse<{}>>(
-      axiosInstance,
-      {
-        method: 'post',
-        url: resourcePath,
-        data: { items },
-      },
+    const chunks = doChunking<RequestParams>(items, chunkFunction);
+    const responses = await promiseAllWithData(
+      chunks,
+      input =>
+        rawRequest<ItemsResponse<{}>>(axiosInstance, {
+          method: 'post',
+          url: resourcePath,
+          data: { items: input },
+        }),
       true
     );
-    return metadataMap.addAndReturn({}, response);
+    return metadataMap.addAndReturn({}, responses[0]);
```

Comment:
The same here, I would merge responses to see an actual result

Reply:
This also returns an empty response...

Reply:
same, what if it did return something once? shouldn't we better know that something went wrong way?

Reply:
https://github.com/cognitedata/cognitesdk-js/pull/176#discussion_r298488697

Reply:
It also unnecessary makes the login in our code more complex by doing that when it is clear from the API that is returns empty results.